gradle-language-server/src/main/java/com/microsoft/gradle/utils/GradleUtils.java

```diff
@@ -0,0 +1,24 @@
+/*******************************************************************************
+ * Copyright (c) 2021 Microsoft Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *    Microsoft Corporation - initial API and implementation
+ *******************************************************************************/
+
+package com.microsoft.gradle.utils;
+
+import org.codehaus.groovy.syntax.SyntaxException;
+import org.eclipse.lsp4j.Position;
+import org.eclipse.lsp4j.Range;
+
+public class GradleUtils {
```

Comment:
About the class name, suggest to move all conversion between LSP item and java element to something like "LSPUtils".